nightingale_msgs/CMakeLists.txt

```diff
@@ -51,11 +51,10 @@ find_package(catkin REQUIRED COMPONENTS
 ##   * add every package in MSG_DEP_SET to generate_messages(DEPENDENCIES ...)
 
 ## Generate messages in the 'msg' folder
-# add_message_files(
-#   FILES
-#   Message1.msg
-#   Message2.msg
-# )
+add_message_files(
+  FILES
+  Payload.msg
```

Comment:
@le2tang i wont put these payload changes as part of the v0 testing PR bc they were not part of v0. feel free to make seperate PR